### PR TITLE
New design that's just about open source

### PR DESCRIPTION
**Pull request by davetron5000**
Will send an email to interested parties, but this is a very simple design that surfaces our current open-source projects only, and more or less matches the design on multi-threaded.

* The projects are auto-generated from GitHub using repo meta-data
* They won't refresh unless we force a rebuild
* This is missing the text for the code of conduct (/cc @codevelvet if you have text for this, you can place it here)

stitch_fix_technology
stitch_fix_technology_and_inbox__9__-_davec_naildrivin5_com_-_naildrivin__5_mail

stitch_fix_technology_and_stitchfix_stitchfix_github_io

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -1,8 +1,52 @@
...
+    <meta name="description" content="{{site.description}}">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="/assets/css/styles.css">
+    <link rel="stylesheet" href="https://cdn.rawgit.com/konpa/devicon/89f2f44ba07ea3fff7e561c2142813b278c2d6c6/devicon.min.css">
+  </head>
+  <body class="container">
+    <header class="text-c mt3">
+      <h1 class="mb1">Stitch Fix Open Source</h1>
+      <p>
+        Open Source software is a huge part of our success.  This site is our small contirbution back.
```
Comment (nickflux) [inline _layouts/default.html]: Typo 'contirbution'

**Comment by nickflux** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Our blog and tech site has moved to http://multithreaded.stitchfix.com  Please follow us there and
 subscribe to theb log.
```
Comment (geoff2k) [inline README.md]: s/theb log/the blog/ -- not your typo, can we fix while we're here?

Comment (jonathandean) [inline README.md]: I was hoping *theb log* was our new branding

*Jon Dean*
Principal Engineer
*Stitch Fix | **your partner in personal style*

*technology.stitchfix.com <http://technology.stitchfix.com>*

jon@stitchfix.com
412-596-8348 (Eastern Time)

File: _layouts/default.html [hunk 3]
```diff
@@ -1,8 +1,52 @@
...
+    <meta name="description" content="{{site.description}}">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="/assets/css/styles.css">
+    <link rel="stylesheet" href="https://cdn.rawgit.com/konpa/devicon/89f2f44ba07ea3fff7e561c2142813b278c2d6c6/devicon.min.css">
+  </head>
+  <body class="container">
+    <header class="text-c mt3">
+      <h1 class="mb1">Stitch Fix Open Source</h1>
+      <p>
+        Open Source software is a huge part of our success.  This site is our small contribution back.
```
Comment (simeonwillbanks) [inline _layouts/default.html]: How about *"These projects are our small contribution back"* (or something similar)?  The projects are more useful than this site.  :wink:

Comment (davetron5000) [inline _layouts/default.html]: Good call.  Fixed in 599dd90808226eb982f6c4e43b8d26f2f36185ea

**Pull request merged**